projects/common/src/telemetry/user-telemetry.module.ts

```diff
@@ -0,0 +1,45 @@
+import { ErrorHandler, Inject, ModuleWithProviders, NgModule } from '@angular/core';
+import { TelemetryGlobalErrorHandler } from './error-handler/telemetry-global-error-handler';
+import { UserTelemetryRegistrationConfig, USER_TELEMETRY_PROVIDER_TOKENS } from './telemetry';
+import { TrackDirective } from './track/track.directive';
+import { UserTelemetryInternalService } from './user-telemetry-internal.service';
+
+@NgModule({
+  declarations: [TrackDirective],
+  exports: [TrackDirective],
+  providers: [
+    {
+      provide: USER_TELEMETRY_PROVIDER_TOKENS,
+      useValue: [],
+      multi: true
+    },
+    {
+      provide: ErrorHandler,
```

Comment:
Because I am injecting ErrorHandler here and plan to import the same module at places where I plan to use `htTrack`, I think I should have `forRoot()` and `forChild()` methods.

Reply:
Probably I should just move this to L31 and rename `withProviders` to `forRoot()`

Reply:
Or just split the directive into a separate module - probably more straight forward.

Reply:
What would we call the new module?

Reply:
TelemetryTrackModule or something like that? Same as our other single component modules.